tests/resolution/evaluation/test_evaluate_operating_system.py

```diff
@@ -29,12 +26,6 @@ async def test_evaluation(coresys: CoreSys):
     assert operating_system.reason not in coresys.resolution.unsupported
     coresys.os._available = False
```

Comment:
This assignment is redundant as it's default. Maybe we should instead add an assert checking OS is not available before the check for unsupported reason.